source/administration/important-upgrade-notes.rst

```diff
@@ -7,6 +7,9 @@ Important Upgrade Notes
 +----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | If you’re upgrading from a version earlier than... | Then...                                                                                                                                                          |
 +====================================================+==================================================================================================================================================================+
+| v5.21.0                                            | Honour key value expiry in KVCompareAndSet, KVCompareAndDelete and KVList. We also improved handling of plugin key value race conditions and deleted keys in     |
```

Comment:
@amyblais, wondering what kind of additional detail to include here. Do we describe the backstory of the bug in question, or just link out to the relevant ticket?

Reply:
I was thinking adding a reason why it's a breaking change and which users it affects. I'm not sure though if these are valuable details to add here.

Reply:
Ah, strictly speaking, this is not a breaking change, relative to the intended semantics -- apologies if I miscommunicated on that front. I was thinking we'd just include this in the release notes.

Reply:
Ok, sounds good.